.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,6 +1,9 @@
 blank_issues_enabled: false
 
 contact_links:
+  - name: Request a new feature for a formula
```

Comment:
```suggestion
  - name: Request a change to a formula
```
maybe?

Reply:
Sure, I wanted to use the phrase "feature request" but I guess that's in the description too.